### ISSUE TITLE
Tuning des Algorithmus (Library 2)

### ISSUE THREAD

**Issue by lukas-monet** [amosproj/amos2023ws03-gui-frame-diff#101]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* Criterion 1
* Criterion 2
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by a-miscellaneous** [comment]
![Image](https://github.com/amosproj/amos2023ws03-gui-frame-diff/assets/96189996/cec97704-0e1f-4bb6-94d6-09e3f85318da)
![Image](https://github.com/amosproj/amos2023ws03-gui-frame-diff/assets/96189996/717d6724-b39f-43b6-a196-9a54ec379646)
